src/main/java/org/folio/service/mapping/writer/impl/JsonRecordWriter.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.service.mapping.writer.impl;
+
+import org.marc4j.MarcJsonWriter;
+import org.marc4j.MarcWriter;
+
+import java.io.ByteArrayOutputStream;
+import java.io.OutputStream;
+
+public class JsonRecordWriter extends MarcRecordWriter {
```

Comment:
The hierarcy seems off here, why does a JsonREcordWriter extend a MarcREcordWriter?

Reply:
The hierarchy is right is this case, because both XmlRecordWriter and JsonRecordWriter extend marc4j mechanisms to build record :)